packages/flutter/test/material/button_theme_test.dart

```diff
@@ -325,4 +326,69 @@ void main() {
     expect(fooText, findsNWidgets(2));
     expect(tester.getRect(fooText.at(0)), tester.getRect(fooText.at(1)));
   });
+
+  testWidgets('button theme with stateful color changes color in states', (WidgetTester tester) async {
```

Comment:
This test appears to be about checking the a11y contrast guidelines

Probably should verify that the specified colors are flowing through instead.

Reply:
Good call, done.